aboutus/tests.py

```diff
@@ -1,3 +1,47 @@
+import json
+from django.core.exceptions import ValidationError
+from rest_framework import status
 from django.test import TestCase
+from factory import DjangoModelFactory
 
-# Create your tests here.
+from aboutus.models import AboutUs
+
+
+class AboutUsFactory(DjangoModelFactory):
+    class Meta:
+        model = AboutUs
+
+    payload_kg = 'Example payload in kyrgyz'
+    payload_ru = 'Example payload in russian'
+
+
+class AboutUsModelTest(TestCase):
+    def test_should_raise_validate_error_if_aboutus_already_exist(self):
+        first = AboutUs.objects.create(payload_kg='First about us', payload_ru='First about us')
+        second = AboutUs.objects.create(payload_kg='Second about us', payload_ru='Second about us')
```

Comment:
Have you tried to use `AboutUsFactory` for this test?

Reply:
No, but I think I should use.